pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -106,4 +108,13 @@ toPythonModule (python.stdenv.mkDerivation (builtins.removeAttrs attrs [
     platforms = python.meta.platforms;
     isBuildPythonPackage = python.meta.platforms;
   } // meta;
-}))
+}));
+
+passthru = {
+  updateScript = let
+    filename = builtins.head (lib.splitString ":" self.meta.position);
+  in writeScript "update-python" ''
+    ${update-python-libraries} ${filename} --commit
```

Comment:
will have to disable committing in case we run concurrently with https://github.com/NixOS/nixpkgs/pull/50977

Reply:
I think it would be nice if `update.py` handled the committing. Though in my experience git’s locking is pretty bad for things like this.

Reply:
Yes, I agree.